Example/RedirectViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RedirectViewController.swift
+//  Example
+//
+//  Created by Bob Godwin Obi on 10/25/17.
+//  Copyright © 2017 RxSwift Community. All rights reserved.
+//
+
+import UIKit
+import WebKit
+import RxWebKit
+import RxSwift
+import RxCocoa
+
+class RedirectViewController: UIViewController {
+    
+    let bag = DisposeBag()
+    let wkWebView = WKWebView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.addSubview(wkWebView)
+        wkWebView.load(URLRequest(url: URL(string: "http://www.webconfs.com/http-header-check.php")!))
+        
+        wkWebView.rx
+            .didReceiveServerRedirectForProvisionalNavigation
+            .debug("didReceiveServerRedirectForProvisionalNavigation")
+            .subscribe(onNext: {(webView, navigation) in
+                let alert = UIAlertController(title: "Redirect Navigation", message: "you have bene redirected", preferredStyle: .alert)
+                alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
+                self.present(alert, animated: true, completion: nil)
```

Comment:
Not super critical but I think you might want to have a `observeOn(MainScheduler.instance)` since you're doing UI logic inside the subscribe block